masterlist.yaml

```
@@ -4887,6 +4887,18 @@ plugins:
       - Graphics
       - Stats
 
+  - name: 'Torch Brightness Radius.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/5101/' ]
+    after:
+      - 'ELE_SSE.esp'
+      - 'ELFX - Hardcore.esp'
+      - 'Lux.esp'
+      - 'Luminosity Lighting Overhaul.esp'
+      - 'RLO - Interiors.esp'
+    tag:
+      - Graphics
+      - Stats
```

The **Stats** tag is unnecessary as it only edits the radius & fade value of two light records. Also, you could add the **Deactivate** tag so that the plugin is deactivated when imported into a Bashed Patch, allowing Wrye Bash to import only the relevant changes:

```suggestion
    tag:
      - Deactivate
      - Graphics
```